<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RLTrader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="main_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Main</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QFrame" name="frame">
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="PageDnButton">
                   <property name="maximumSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="resources.qrc">
                     <normaloff>:/icons/icons/skip_previous_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icons/skip_previous_FILL0_wght400_GRAD0_opsz48.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="PageUpButton">
                   <property name="maximumSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="resources.qrc">
                     <normaloff>:/icons/icons/skip_next_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icons/skip_next_FILL0_wght400_GRAD0_opsz48.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="ticks_frame_chart_layout"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionRecalculate_ticks"/>
    <addaction name="actionRecalculate_ticks_percents"/>
    <addaction name="actionRecalculate_ticks_points"/>
    <addaction name="actionRecalculate_zigzag"/>
    <addaction name="actionRecalculate_EMA"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommands"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionRecalculate_ticks">
   <property name="text">
    <string>Recalculate &amp;ticks</string>
   </property>
  </action>
  <action name="actionRecalculate_zigzag">
   <property name="text">
    <string>Recalculate &amp;zigzag</string>
   </property>
  </action>
  <action name="actionRecalculate_EMA">
   <property name="text">
    <string>Recalculate &amp;EMA</string>
   </property>
  </action>
  <action name="actionRecalculate_ticks_percents">
   <property name="text">
    <string>Recalculate ticks &amp;percents</string>
   </property>
  </action>
  <action name="actionRecalculate_ticks_points">
   <property name="text">
    <string>Recalculate ticks p&amp;oints</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
